eth/tracers/firehose.go

```diff
@@ -186,6 +190,11 @@ type Firehose struct {
 	// Testing state, only used in tests and private configs
 	testingBuffer             *bytes.Buffer
 	testingIgnoreGenesisBlock bool
+
+	// Worker queue for block printing
+	blockPrintQueue chan *blockPrintJob
+	flushDone       sync.WaitGroup
```

Comment:
Let's prefix with `block` so `blockFlushDone` to make the link it's block related, move to `// Block state` section like other `block` prefixed variables.